### PR TITLE
Remove define plugin and use default paths which will be proxied

### PR DESCRIPTION
**Pull request by karelhala**

### CODE REVIEW

File: frontend/config/plugins.js [hunk 1]
```diff
@@ -25,16 +25,6 @@ const plugins = [
   new CopyWebpackPlugin({
     patterns: [{ from: 'static' }],
   }),
-  new DefinePlugin({
-    K8S_TARGET_URL: JSON.stringify(
```
Comment (florkbr) [inline frontend/config/plugins.js:29]: @karelhala if these are no longer used, can we remove the `globals` entries for them in `.eslintrc.js` and `index.d.ts`?

Comment (florkbr) [reply frontend/config/plugins.js:29]: Also wanted to confirm we did not need any additional changes on Akamai before merging this PR.

Comment (karelhala) [reply frontend/config/plugins.js:29]: Mhh, good point. We are not using WS when deployed, however we should sync this with Akamai change to add the proxy in there to have the k8s ws test available.

**Comment by karelhala** [comment]
@florkbr the upate of proxy path on Akamai has been deployed. We are good to go with this one.

**Comment by florkbr** [comment]
@psav @karelhala my original comment was never addressed here...
https://github.com/openshift/hac-core/pull/56#discussion_r845550598

**Comment by karelhala** [comment]
@florkbr ah, shoot! Didn't notice it :disappointed:

**Pull request merged**